### PR TITLE
[13.0][ADD]partner_analytic

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
Adds Income and Expense Analytic Accounts for the Contacts for then to be set as default in Invoices / Bills.

cc @ForgeFlow

### CODE REVIEW

**Review by mariadforgeflow — APPROVED** [review]
Good work! LGTM

**Review by JoanSForgeFlow — APPROVED** [review]
LGTM!

File: partner_analytic/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        ):
+            an_accs = partner.get_partner_analytic_accounts()
+            an_acc = an_accs[INV_TYPE_MAP[inv_type]]
+            self.analytic_account_id = an_acc
+        return res
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if "move_id" in vals:
```
Comment (JordiMForgeFlow) [inline partner_analytic/models/account_move.py]: To make it more readable you could first do if "move_id" not in vals: continue, and then do the rest

Comment (GuillemCForgeFlow) [reply partner_analytic/models/account_move.py]: Thank you 👍🏽

File: partner_analytic/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        string="Customer Analytic Account",
+        help="Default Analytic Account used for Invoices / Credit Notes.",
+    )
+
+    property_supplier_analytic_account_id = fields.Many2one(
+        comodel_name="account.analytic.account",
+        company_dependent=True,
+        string="Vendor Analytic Account",
+        help="Default Analytic Account used for Vendor Bills / Vendor Refund.",
+    )
```
Comment (JordiMForgeFlow) [inline partner_analytic/models/res_partner.py:22]: to align them with other account fields in partner we should make them commercial fields to only be managed at the parents

Comment (JordiMForgeFlow) [inline partner_analytic/models/res_partner.py:22]: to align them with other account fields in partner we should make them commercial fields to only be managed at the parents

Comment (GuillemCForgeFlow) [reply partner_analytic/models/res_partner.py:22]: done!

**Review by JordiMForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LG

**Pull request merged**